src/app/core/store/selectors/defendants.selectors.ts

```diff
@@ -44,8 +44,13 @@ export const getAllDefendantsForAssetId = (aAssetId: number) => createSelector(
 
 export const getDefendantsLoaded = createSelector(getDefendantsState, fromDefendants.getDefendantsLoaded);
 export const getDefendantsLoadedForAssetId = (aAssetId: number) => createSelector(
-  getDefendantsLoaded,
-  (aLoaded) => aLoaded[aAssetId] || false
+  getDefendantsState,
+  (defendantsState) => {
+    if (aAssetId in defendantsState.assetsToDefendantsCount) {
+      return defendantsState.loaded[aAssetId] || [];
```

Comment:
`getDefendantsLoadedForAssetId` should always return a boolean.

Reply:
reverted to previous state since it was not needed